src/pages/guides/using/aem/cloud-native/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: AEM Events as a Cloud Service
+description: Introduction to the all-new AEM Events in AEM as a Cloud Service
```

Comment:
@peterklassen  Please can you review Wording 'AEM Events in AEM as a Cloud Service' it lokks really long and not very developer friendly. cc @shikhartanwar